test/IntegrationTests/SmokeTests.cs

```diff
@@ -39,7 +39,6 @@ public SmokeTests(ITestOutputHelper output)
         : base("Smoke", output)
     {
         SetEnvironmentVariable("OTEL_SERVICE_NAME", ServiceName);
-        SetEnvironmentVariable("OTEL_DOTNET_AUTO_TRACES_ENABLED_INSTRUMENTATIONS", "HttpClient");
```

Comment:
I would leave this one, as probably, we would have no integration test for `OTEL_DOTNET_AUTO_TRACES_ENABLED_INSTRUMENTATIONS` at all. If there is some test that checks `OTEL_DOTNET_AUTO_TRACES_ENABLED_INSTRUMENTATIONS` then ignore this comment.